src/Playwright/Transport/Channels/FrameChannel.cs

```diff
@@ -700,5 +701,29 @@ internal Task DragAndDropAsync(string source, string target, bool? force, bool?
 
             return Connection.SendMessageToServerAsync(Guid, "dragAndDrop", args);
         }
+
+        internal async Task<FrameExpectResult> ExpectAsync(string selector, string expression, object expressionArg, ExpectedTextValue[] expectedText, int? expectedNumber, object expectedValue, bool? useInnerText, bool? isNot, float? timeout)
+        {
+            var args = new Dictionary<string, object>
+            {
+                ["selector"] = selector,
```

Comment:
this could be using `[nameof(selector)] = selector` if you plan to keep the parameter names consistent with those keys.

Reply:
I'll keep it as it is for now, since it's then more clear. But interesting suggestion!